### DISCUSSION TITLE
How to change Python version the customtkinter gets installed?

### DISCUSSION THREAD

**Discussion by floprete09** [TomSchimansky/CustomTkinter#311] [Q&A]
I'm new to Python and tried to install customtkinter but when I installed it it was installed in my python 3.6 instead of my python 3.9 that I have been using. I can switch to 3.6 on VsCode, but my other things don't work then that I have installed for Python3.9. I would like to just get rid of Python3.6 but not sure how but that's another issue. My system is running Linux Mint. I tried to simply locate the folder for customtkinter and move it to the appropriate folder for Python3.9, and I found the customtkinter folder for python3.6  but I can't find the proper directory to put it in python3.9. I I tried one that I found, but it didn't work. Vscode still errors on me when I run my program. Python3.9 is not installed in the same path as 3.6 and python2.7 on my computer and idk where it is or if copying and pasting the customtkinter folder will even work. Anyway, I just want to install customtkinter in the python3.9 folder. How do I do that? 

![Screenshot from 2022-07-22 13-26-42](https://user-images.githubusercontent.com/94071026/180690736-8a648c74-9499-4013-8419-928a13f44672.png)
![Screenshot from 2022-07-22 13-28-50](https://user-images.githubusercontent.com/94071026/180690745-ad5de023-41b7-4c8c-ba9d-f18de02d3382.png)
![Screenshot from 2022-07-22 15-12-54](https://user-images.githubusercontent.com/94071026/180690748-82677e57-4046-417d-b6ce-e52f4498d68b.png)

**Comment by ArielMAJ** [comment]
Well, taking into account you're a beginner, I'd just say you're gonna have some headaches sometimes trying to solve things that feels really hard to solve. But that's life and you'll get through it.

I'd recommend you to keep trying unninstalling python 3.6 and try to stick only to the versions you're really using.

An easy work around that will probably work (for ctk at least) but I don't recommend sticking to it for too long (try to pip install it properly later) is:
1. Git clone or download ctk folder.
2. Put the customtkinter folder somewhere inside the project you're using it.
3. Import it.

This will work only for the project that is using it and isn't the best solution.

**Comment by TomSchimansky** [comment]
How do you installed customtkinter? So if you have two python versions installed, you should be able to start them either with `python3.6 <file_to_run.py>` or `python3.9 <file_to_run.py>`. And if you install modules you should also have two different `pip` commands. If you want to install a module for 3.6 you do: `pip3.6 install <module>` and for 3.9 you do `pip3.9 install <module>`.
And CustomTkinter only works with python>=3.7.

**Comment by plugeit** [comment]
Go to python3.6/Lib/site_packages and then copy cutomtkinter.
![image](https://user-images.githubusercontent.com/72407828/181029324-0a184ac0-2ec2-4d8c-b761-2e0ed443fbaa.png)

**Reply by TomSchimansky** [reply]
I would be very careful to copy folders around in the library folders, it's not necessary and you should let the package manager do it for you.

**Reply by ArielMAJ** [reply]
Yeah, sometimes it may work, but it won't for a few libraries (plus you may run into unexpected bugs and not realize the reason).

**Comment by felipetesc** [comment]
If you are only interested right now to use github customtkinter version and you don't want to install using pip you can do this:
```python
import sys
sys.path.append("/location/to/the/library/called/CustomTkinter/")
```
Just call this sample code inside the main before importing customtkinter and don't forget to change to the correct path

**Comment by avalon60** [comment]
@floprete09, you might also want to look at virtual environments. Python lets you sandbox your projects. Things can get messy and confusing) if you don't use them. You can produce them from the command line or a good IDE should allow for this. 
Something like this might help:
https://pythonbasics.org/virtualenv/
